src/main/groovy/com/blackducksoftware/integration/hub/detect/util/executable/ExecutableManager.groovy

```diff
@@ -61,35 +61,42 @@ class ExecutableManager {
         }
     }
 
-    String getExecutableName(ExecutableType executableType) {
-        executableType.getExecutable(currentOs)
+    String getDefaultExecutableName(ExecutableType executableType) {
+        executableType.getExecutables(currentOs)[0]
```

Comment:
What makes something a default? It seems arbitrary to get the first element (because this change clearly outlines the validity of multiple formats) when in ExecutableType.groovy, MVN and MVNW have the same list for WINDOWS, but in a different order. It is already a breaking change, so why not just scrap this method?

Reply:
Good catch, that was a remnant of refactoring out the old method.